release-notes/opensearch-dashboards.release-notes-1.3.0.md

```diff
@@ -396,7 +396,7 @@
     
     
     Clean up temp aws paths in code base. Add a configurable flag
-    `showRegionBlockedWarning` into map plugin level config file.
+    `showRegionDeniedWarning` into map plugin level config file.
```

Comment:
I think for release note, we don't need to clean them to avoid confusion. We could specify what are deprecated in this PR's commit msg.